OptiTypePipeline.py

```diff
@@ -139,10 +139,15 @@ def get_types(allele_id):
             return table.loc[aa[0]]['4digit']
         else:
             return table.loc[aa[0]]['4digit']  #+ '/' + table.loc[aa[1]]['4digit']
-            
-            
-            
-            
+
+def get_num_threads(configured_threads):
+  try:
+    import psutil
```

Comment:
Could you perhaps use multiprocessing instead so that we do not have add another dependency